test/fixtures/openssl_test_engine/node_test_engine.c

```diff
@@ -0,0 +1,14 @@
+#include <openssl/engine.h>
+
+static const char *engine_id = "node_test_engine";
+static const char *engine_name = "Node Test Engine for OpenSSL";
```

Comment:
Tiny style nit: star leans left (`const char* engine_id = ...`.).  You could also writes this as:
```c
static const char engine_id[] = ...;
```
Saves a pointer indirection.